ansible/roles_ocp_workloads/ocp4_workload_logging/files/logging_curator_configmap.yaml

```diff
@@ -62,7 +63,7 @@ data:
       delete:
         days: 5
   curator5.yaml: "---\nclient:\n  hosts:\n  - ${ES_HOST}\n  port: ${ES_PORT}\n  use_ssl:
-    True\n  certificate: ${ES_CA}\n  client_cert: ${ES_CLIENT_CERT}\n  client_key:
-    ${ES_CLIENT_KEY}\n  ssl_no_validate: False\n  timeout: ${CURATOR_TIMEOUT}\n  master_only:
-    False\nlogging:\n  loglevel: ${CURATOR_LOG_LEVEL}\n  logformat: default\n  blacklist:
+    true\n  certificate: ${ES_CA}\n  client_cert: ${ES_CLIENT_CERT}\n  client_key:
+    ${ES_CLIENT_KEY}\n  ssl_no_validate: false\n  timeout: ${CURATOR_TIMEOUT}\n  master_only:
+    false\nlogging:\n  loglevel: ${CURATOR_LOG_LEVEL}\n  logformat: default\n  blacklist:
```

Comment:
Let's get this cleaned up and reformatted with a `|` quote operator...

Looks like it should be:

```
  curator5.yaml: |
    ---
    client:
      hosts:
      - ${ES_HOST}
      port: ${ES_PORT}
      use_ssl: true
      certificate: ${ES_CA}
      client_cert: ${ES_CLIENT_CERT}
      client_key: ${ES_CLIENT_KEY}
      ssl_no_validate: false
      timeout: ${CURATOR_TIMEOUT}
      master_only: false
    logging:
      loglevel: ${CURATOR_LOG_LEVEL}
      logformat: default
      blacklist:
      - elasticsearch
      - urllib3
```